py3status/modules/uptime.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+
+"""
+Display the system's uptime
+
+Configuration parameters:
+        - format : what will be displayed, formatted with predefined
+          placeholders; defaults to "{days} days, {hours} hours, {minutes} min"
+
+Available placeholders in format string:
+        - {days}: days part of the uptime
+        - {hours}: hours part of the uptime
+        - {minutes}: minutes part of the uptime
+        - {seconds}: seconds part of the uptime
+        - No {year} provided !
```

Comment:
> I think you should either add a year or remove the this is not an option
> good hygiene suggests a year uptime should be avoided

Considering the fact the py3status would be ran on machines with a WM, as pointed by @pydsigner, I'm not really up to add a `{year}` placeholder. I also agree with imitating the standard `uptime` command which doesn't represent years.

I wanted to point the fact that the `{year}` placeholder wasn't available so people don't try to use it, but if I "enable" it based on format... why not !